plugin/src/main/kotlin/me/gabber235/typewriter/entry/EntryDatabase.kt

```diff
@@ -22,6 +26,8 @@ object EntryDatabase {
 		private set
 	private var dialogue = listOf<DialogueEntry>()
 	private var actions = listOf<ActionEntry>()
+	// used for registering commands
+	private var commandEvents = listOf<EventEntry>()
```

Comment:
From the previous comment, if you make a custom interface for commands this can change to `CustomCommandEntry`